### PR TITLE
ignore: revise with cats example data

### PR DESCRIPTION
**Pull request by mcritchlow**
Closes: #81 

changes:
- integrate the files from cats-as-data
- remove a lot of the extras on ignoring things

### CODE REVIEW

File: _episodes/05-ignore.md [hunk 1]
```diff
@@ -34,40 +31,29 @@ On branch master
...
 ~~~
-$ nano .gitignore
-$ cat .gitignore
-~~~
-{: .bash}
-
-~~~
-*.dat
-results/
+# ...
```
Comment (dunn) [inline _episodes/05-ignore.md]: What does this bit mean?  Do we even need this code block?

Comment (mcritchlow) [reply _episodes/05-ignore.md]: Oh, that was to just show putting the line for the file to ignore below the comments. I could just remove that. But I think it's worth showing the line they have to add with the filename. Just to remove any potential confusion about whether they need to put any kind of path in, quote it, etc.

File: _episodes/05-ignore.md [hunk 2]
```diff
@@ -34,40 +31,28 @@ On branch master
...
 ~~~
-$ nano .gitignore
-$ cat .gitignore
-~~~
-{: .bash}
-
-~~~
-*.dat
-results/
+cats-human-situations.md
```
Comment (mcritchlow) [inline _episodes/05-ignore.md]: @dunn - better? i can see how the `#...` might have confused ppl.

File: _episodes/05-ignore.md (hunk 1 above)
Comment (dunn) [reply _episodes/05-ignore.md]: Oh, that's the contents of `.gitignore`.  I guess that could be more explicit.

File: _episodes/05-ignore.md (hunk 2 above)
Comment (dunn) [reply _episodes/05-ignore.md]: Can you add something above like, 'this is what the file should contain'?

Comment (mcritchlow) [reply _episodes/05-ignore.md]: Isn't that already in the line above?

Comment (dunn) [reply _episodes/05-ignore.md]: I guess it would be if the aforementioned comments were also added.

**Review by chrissyrissmeyer — APPROVED** [review]
looks good to me

File: _episodes/05-ignore.md (hunk 2 above)
Comment (mcritchlow) [reply _episodes/05-ignore.md]: Yeah that's a good point. Added them in. Thanks!

**Review by dunn — APPROVED** [review]

**Pull request merged**